packages/brand-feed-luigis-box/src/Resources/views/feed.xml.twig

```diff
@@ -0,0 +1,17 @@
+{% block begin %}
+<?xml version="1.0" encoding="utf-8"?>
+<brands>
+{% endblock %}
+
+{% block item %}
+    <brand>
+        <identity>{{ item.identity }}</identity>
+        <name>{{ item.name }}</name>
+        <url>{{ item.url }}</url>
+        <image_url>{{ item.imageUrl }}</image_url>
```

Comment:
when imageUrl is null, is it ok to render empty tag?

Reply:
Nice catch, thx